src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,61 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+import java.util.ArrayList;
+
+/**
+ * Represents addition or deletion of tag for a person during a session
+ *
+ * Used for displaying tags changed at the end of a session
+ */
+
+public class Tagging {
+    private static ArrayList<Tagging> taggings;
+
+    public enum CHANGE {
+        ADD, DELETE
+    }
+
+    private final Person person;
+    private final Tag tag;
+    private final CHANGE change;
+
+    /**
+     * @param person person whose tag was changed
+     * @param tag tag that was changed
+     * @param change type of change
+     */
+    public Tagging(Person person, Tag tag, CHANGE change) {
+        this.person = person;
+        this.tag = tag;
+        this.change = change;
+    }
+
+    /**
+     * Adds tagging to class-level variable that stores all taggings
+     */
+    public static void addTagging(Tagging tagging) {
```

Comment:
This forces the caller to manually call `addTagging` after creating a new `Tagging`. If he doesn't, the doc comment for `getTaggings` will be incorrect. One way to solve this would be to move this method into the constructor. Alternatively, a new singleton `TaggingList` class can be created.